src/Bundle/DependencyInjection/SyliusGridExtension.php

```diff
@@ -77,4 +75,97 @@ public function getConfiguration(array $config, ContainerBuilder $container): Co
 
         return $configuration;
     }
+
+    private function loadPersistence(array $drivers, array $grids, LoaderInterface $loader, ContainerBuilder $container): void
+    {
+        $availableDrivers = $this->getAvailableDrivers($container);
+
+        // Enable all available drivers if there is no configured drivers
+        $drivers = [] !== $drivers ? $drivers : $availableDrivers;
+
+        $gridDrivers = $this->getGridDrivers($grids);
+
+        $this->checkConfiguredDrivers($drivers, $availableDrivers, $gridDrivers);
```

Comment:
This code is a bit hard to understand, I've left previous comments to illustrate it, but here is a resume.

```php
getAvailableDrivers()
resolveDriversUsedInGrids()
assertDriversValid()
foreach ($drivers in $driver)
...
```